ci/semaphore-jobs.sh

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2017 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#!/bin/bash
+
+set -e
+
+go_packages=$(go list ./... | grep -v vendor)
+
+echo -e "go test $go_packages -cover"
+sudo -E go test $go_packages -cover
```

Comment:
Since your log statements are identical to the commands being run, I'd go for one of these two approaches:

- Use `eval`
  ```
  cmd="sudo go test $go_packages -cover"
  echo "$cmd"
  eval $cmd
  ```
- Use `set -x`
  Remove all the `echo`'s and add `set -x` at the top of the script.